src/cpp/r/session/graphics/RGraphicsDevice.cpp

```diff
@@ -651,7 +650,7 @@ Error restoreSnapshot(const core::FilePath& snapshotFile)
 void copyToActiveDevice()
 {
    int rsDeviceNumber = GEdeviceNumber(s_pGEDevDesc);
-   GEcopyDisplayList(rsDeviceNumber);
```

Comment:
I noticed that this was called without an `r::exec::executeSafely()`; unfortunately adding that wrapper was insufficient to catch the error on Windows.